go.mod

```diff
@@ -101,11 +101,8 @@ require (
 )
 
 replace (
-	// https://nvd.nist.gov/vuln/detail/CVE-2022-1996
-	github.com/emicklei/go-restful => github.com/emicklei/go-restful v2.16.0+incompatible
-
-	// https://avd.aquasec.com/nvd/2022/cve-2022-27191/
-	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220314234659-1baeb1ce4c0b
+	// fix CVE-2023-47108 introduced by k8s.io/apiextensions-apiserver
+	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc => go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.46.0
 
 	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.26.1 // weird bug that the goland won't compile without this
```

Comment:
what about this one?

Reply:
I tried to remove this line, but the goland would complain that the k8s.io/kube-scheduler@v0.0.0 is invalid.

I think that's the bug mentioned in the comment which i cannot resolve.